app/javascript/state/experiments/index.js

```diff
@@ -0,0 +1,23 @@
+export const SET_VARIANT = 'set_variant';
+
+const initialState = {
+  experiments: [],
```

Comment:
Using an array in case we have several experiments running at the same time in the future